docs/offline_wallets.md

```diff
@@ -0,0 +1,86 @@
+# Offline wallets
+
+Cold wallets may be monitored using a watching only wallet. A watching only 
+wallet is created using an extended public key for an account.
+
+An extended key for your cold wallet can be retrieved using the 'getmasterpubkey'
+command in the legacy API. Without any argument it will return the default 
+account extended public key. Other accounts may be retrieved by supplying the 
+account as an argument.
+
+A wallet is then created using the public key, by the following command:
+
+```
+dcrwallet --create --createwatchingonly
+```
+
+This wallet can safely be connected to an online daemon and used to monitor the 
+cold wallet. It can be used to get new addresses and supply a UTXO list with 
+the command 'listunspent'.
+
+A tool has been created to help easily spend offline funds on *nix machines. 
+This tool is located in cmd/movefunds and can be installed as follows:
+
+```
+cd $GOPATH/src/github.com/dcrwallet/cmd/movefunds
+go install
+```
+
+You may also have to install jq on the cold wallet machine. For debian-based 
+builds, you can use apt-get or build yourself.
+
+```
+sudo apt-get install jq
+```
+
+To move money from the cold wallet without having to connect to the network, 
```

Comment:
Also might want to to move funds from the cold wallet to a hot wallet or something like that just to be clearer (since there are 3 wallets in this example, the wow, the cold, and a hot one).